pages/Alert/alert.css

```diff
@@ -0,0 +1,304 @@
+
+*{
+      margin: 0;
+      padding: 0;
+}
+:root {
+      --primary-color: #F94892;
+      --secondary-color: #323232;
+      --black-color: #3f3c3b;
+      --white-color: #F9F9F9;
+  }
+
+.container{
+      height:100vh;
+      display: grid;
+      grid-template-columns:300px repeat(2, 1fr);
+      grid-template-rows: 5rem 35rem 5rem;
+      grid-template-areas: 
+      "nav nav nav"
+      "side main main"
+      "footer footer footer"
+      ;    
+}
+.navigation{
+      grid-area: nav;
+      grid-column: 1/-1;
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      height: 100%;
+      background-color: var(--secondary-color);
+      width: 100%;
+}
+.nav-home-page
+  {
+        display: flex;
+        list-style: none;
+        text-decoration: none;
+        font-size: 1.2rem;
+        margin-right: 5rem;
+       
+  }
+  .cl-name{
+        padding-bottom: 1rem;
+      display: flex;
+      flex-direction: column;
+      margin-left: 4rem;
+      font-size: 2rem;
+      color: var(--primary-color);
+      
+}
+.sub-title-name{
+      font-size: 1.1rem;
+      color: var(--white-color);
+        }
+      
+        .sub-title-name:hover{
+              color:var(--primary-color)
+        }
+        .home{
+            color: var(--white-color);
+           padding: 2rem;
+           text-decoration: none;
+    
+    
+      }
+      .home:hover{
+          color: var(--primary-color);
+    }
+
+/* Sidebar code */
+
+.sidebar{
+      grid-area: side;
+      width: 100%;
+      height:99.5%;
+      background-color: var(--secondary-color);
+      border: 2px solid white;   
+}
+.get-started{
+      margin-top: 7rem;
+}
+
+aside{
+    display: flex; 
+      align-items: center;
+      justify-content: center;
+      flex-direction: column;
+      text-decoration: none;
+      list-style: none;
+      color: var(--white-color);
+      font-size: 1.5rem;
+}
+.bullet-none{
+    text-decoration: none;
+    list-style: none;
+    color: var(--white-color);
+   
+   
+}
+aside li{
+    padding: 0.6rem;
+    font-size: 1.2rem;
+
+}
+aside li a{
+    text-decoration: none;
+    color: var(--white-color);
+    
+}
+aside li a:hover{
+    color: var(--primary-color);
+}
+.sidebar{
+    overflow-y:scroll;
+}
+
+/* Main Content */
+
+.main-content{
+      height: 100vh;
+    background-color: var(--black-color);
+      grid-area: main;
+      overflow-y:scroll;
+      }
+.alert-title{
+      margin: 2rem 0 0 5rem;
+      color: var(--primary-color);
+      font-size: 3rem;
+}
+.alert-details{
+      padding: 10px;
+      color: var(--white-color);
+      font-size: 1.3rem;
+      width: 900px;
+      padding-left: 5rem;
+}
+.alert-example{
+      margin: 2rem 0 0 5rem;
+      color: var(--white-color);
+      font-size: 2rem;
+}
+.example-details{
+      padding: 10px;
+      color: var(--white-color);
+      font-size: 1.1rem;
+      width: 900px;
```

Comment:
just a suggestion, using relative units like `rem` would be more useful for you to work with responsiveness